test/behaviour/http-mock.js

```diff
@@ -0,0 +1,8 @@
+
+var mock = require('protractor-http-mock');
+mock.config = {
+	rootDirectory: __dirname, // default value: process.cwd()
+	protractorConfig: 'protractor.conf.js'
```

Comment:
What does this line do? Because for browserstack tests, the protractor config file will be `protractor.browserstack.conf.js`. Could not adapting this maybe cause weird bugs at a later point?